pudb/forked.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+
+import sys
+import fcntl
+import termios
+import struct
+
+from pudb.debugger import Debugger
+
+
+def set_trace(paused=True, frame=None, term_size=None):
+    """Set a breakpoint in a forked process on Unix system, e.g. Linux & MacOS.
+    In- and output will be redirected to /dev/stdin & /dev/stdout.
+    You can call pudb.forked.set_trace() directly or
+    use it with python's built-in breakpoint():
+    PYTHONBREAKPOINT=pudb.forked.set_trace python …
+    """
+    if frame is None:
+        frame = sys._getframe().f_back
+    if term_size is None:
+        try:
+            # Getting terminal size
+            s = struct.unpack(
+                "hh",
+                fcntl.ioctl(1, termios.TIOCGWINSZ, "1234"),
+            )
+            term_size = (s[1], s[0])
+        except Exception:
+            term_size = (80, 24)
+
+    Debugger(
+        stdin=open("/dev/stdin"),
+        stdout=open("/dev/stdout", "w"),
```

Comment:
Should these be opened in binary mode?

Reply:
I wasn't sure and tested it, but using binary mode results in an `io.UnsupportedOperation: File or stream is not seekable.` error.

Reply:
Hmm, thanks for checking. I'm not sure I understand the failure mode, but I'm OK with leaving this be for now.